docs/source/scripts_and_their_functions.rst

```diff
@@ -26,6 +25,33 @@ User Input Forwarder
 --------------------
 This script mimics a distribution node. It listens to documents coming from the User Input Producer on ``ws://127.0.0.1:9001`` and forwards them to any consumer listening on ``ws://127.0.0.1:9000``. Like the Simple Producer, it can also save the documents it receives to the file system. First, run it with the ``--folder-export`` argument like this: ``ebu-user-input-forwarder --folder-export myFolder``. Then launch the User Input Producer and connect. The sequence will be saved to ``myFolder`` along with the manifest file. The User Input Forwarder can also be used as an incoming connection point for WebSocket connections from sources other than the User Input Producer.
 
+Handover Manager
+----------------
+This node implements the 'Who claimed control most recently' algorithm defined in the specification. This algorithm determines the output from multiple input sequences.
+The Handover Manager is a specialised case of the switching node that bases its decisions on handover-related attributes in the document and its previous decisions.
+There is no separate command to run this script. Start it with the ``ebu-run`` script.
+
+To see it in action using the Websocket transport mechanism, follow these steps:
+
+* Start the Handover Manager: ``ebu-run --admin.conf=ebu_tt_live/examples/config/user_input_producer_handover.json``
+
+* Create two (or more) instances of the User Input Producer by opening ``ebu_tt_live/ui/user_input_producer/user_input_producer.html`` in multiple browser tabs/windows.
+
+* In each instance of the UIP, create a sequence with a unique sequence identifier but with identical authors group identifier. If you use the default configuration, this is "TestGroup1".
```

Comment:
change "unique" to "different"? Otherwise it could be read as saying you have to use the same sequence identifier everywhere.